contracts/Loans.sol

```diff
@@ -337,7 +337,7 @@ contract Loans is DSMath {
             require(!sales.taken(sales.salel(loan, sales.next(loan) - 1))); // Can only start auction again if previous auction bid wasn't taken
 		}
 		sale = sales.open(loan, loans[loan].bor, loans[loan].lend, loans[loan].agent, sechi(loan, 'A'), sechi(loan, 'B'), sechi(loan, 'C'), tokes[loan], vares[loan]);
-        require(tokes[loan].transfer(address(sales), back(loan)));
+        if (bools[loan].sale == false) { require(tokes[loan].transfer(address(sales), back(loan))); }
```

Comment:
Ensure that transfer of partial repayments is only sent to `Sales` contract once.